jupyterhub/templates/proxy/deployment.yaml

```diff
@@ -70,6 +71,10 @@ spec:
             - --redirect-to=443
             - --ssl-key=/etc/chp/tls/{{ .Values.proxy.https.secret.key }}
             - --ssl-cert=/etc/chp/tls/{{ .Values.proxy.https.secret.crt }}
+            {{- else if $offloadHTTPS }}
+            - --port=8443
+            - --redirect-port=8000
+            - --redirect-to=443
```

Comment:
What flags should be used here? I figure the goal is really to send back a redirect with a Location header (where someone is redirected) being the same as the request originally but with a https:// scheme.